client/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import { italic } from 'ansi-colors';
+
+italic('renders without crashing', () => {
```

Comment:
What does this function do? Not familiar with the `italic` method and what the first parameter is

Reply:
I copied that test code from a medium article while trying to set up front end tests, not actually sure how it ended up in this branch. This entire function is called a smoke test, just returns true if the app renders correctly

Reply:
Maybe add a comment to clarify this, then?